src/signup-form/us/student.yml

```diff
@@ -0,0 +1,116 @@
+header:
+  title: Student Registration
+  notes: |
+    <p align="center">
+    Would you like to get tutoring? <a class="btn small green" href="/signup-pupil" sveltekit:prefetch>Signup student</a>
+    </p>
+
+    ## Form for students
+
+    Your details will be saved in [Airtable](https://airtable.com). Information on data processing can be found in our [privacy policy](/privacy policy).
+    Please fill out all fields marked with a red star and then click on "Submit registration".
+
+fields:
+  - id: chapter
+    title: Location
+    note: At which location would you like to give tutoring?
+    type: select
+    required: true
+    maxSelect: 1
+
+  - id: gender
+    title: Gender
+    type: select
+    required: true
+    maxSelect: 1
+
+  - id: fullName
+    title: first and last name
+    note: Please enter it as it appears on your ID card. Is required for issuing the free certificate of good conduct.
+    required: true
+
+  - id: phone
+    title: Phone
+    note: Preferably a cell phone number so that we can easily reach you for mediation.
+    type: phone
+
+  - ID: email
```

Comment:
id

Reply:
Why are the ids not `emailContact` (other resp.) like in the german yml?

Reply:
The text was auto translated by janosh. So some formations were also modified